client/task-list/tasks/payments/RemotePayments/components/PaymentConnect.js

```diff
@@ -103,39 +137,45 @@ export const PaymentConnect = ( {
 		},
 	} );
 
-	return (
-		<Form
-			initialValues={ getInitialConfigValues() }
+	const DefaultSettings = ( { ...props } ) => (
+		<Settings
+			fields={ fields }
+			isBusy={ isOptionsRequesting }
 			onSubmitCallback={ updateSettings }
+			onButtonClickCallback={ () => recordConnectStartEvent( key ) }
+			buttonLabel={ __( 'Proceed', 'woocommerce-admin' ) }
 			validate={ validate }
```

Comment:
Just thinking out loud here: I wonder if validation will happen on the input level (individual settings) or form (settings group) level.  I generally prefer on the form level as it is now since it allows validation by comparing multiple fields, but just wondering how that ties into existing validation with WC Settings.

My guess is that we should keep things as you have them here since client-side validation will have to be freshly added anyway, but just putting this out there to think about.

Reply:
Yes, bit of an open question for me as well. I'll keep what I have currently and we may revisit soon.